bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormStructureParser.java

```diff
@@ -43,4 +47,20 @@ public interface FormStructureParser {
      * @return true if this resource or one of its children is a form container, else false
      */
     Boolean containsFormContainer();
+
```

Comment:
Since addClientLibRef doesn't directly relate to the core responsibilities of parsing form structures, it would be better to separate this functionality into a different component. 

Let's do the following,
```
public interface FormClientLibManager {
    void addClientLibRef(String clientLibRef);
    List<String> getClientLibReferences()
}
``
We should instead use this interface to manage all forms client library references rather than polluting forms structure parser whose core responsibility is to just parse form structure.

Reply:
Please add proper java docs for the new interface, update the package-info.java file. Not sure why the maven plugin isn't complaining to bump the version